Libraries/CameraRoll/RCTImagePickerManager.m

```diff
@@ -146,6 +148,16 @@ - (void)imagePickerController:(UIImagePickerController *)picker
     height = @(image.size.height);
     width = @(image.size.width);
   }
+  
+  if (isMovie) {
```

Comment:
prefer we put this as `else if (isMovie) {...}` so called only when `image` is nil and avoid potential override.